### PR TITLE
Adding support for deploying ceph-osd in containers with kv backend

### PR DESCRIPTION
**Pull request by PiotrProkop**

### CODE REVIEW

File: roles/ceph-osd/defaults/main.yml [hunk 1]
```diff
@@ -109,6 +109,9 @@ osd_directory: false
 ##########
 
 osd_containerized_deployment: false
+osd_containerized_deployment_with_kv: false
+kv_type: etcd
+kv_endpoint: 127.0.0.1
```
Comment (leseb) [inline roles/ceph-osd/defaults/main.yml]: I think you're missing `ceph_osd_docker_prepare_env`

**Comment by leseb** [comment]
test this please

**Comment by leseb** [comment]
test this please

**Comment by leseb** [comment]
@PiotrProkop sorry I just merged the docker branch into master, would you mind rebasing your change and propose it to master instead?
Thanks and sorry for the pain.

**Comment by PiotrProkop** [comment]
No problem I'll close this PR and make new one.

**Comment by leseb** [comment]
Thanks!

**Pull request closed without merging**